argilla/src/argilla/_exceptions/_responses.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from argilla._exceptions._base import ArgillaError
+
+
+class ResponsesError(ArgillaError):
```

Comment:
Should we make the name clearer? RecordResponse* or ArgillaResponse* just so it's clear that it's nothing to do with an API response.